activity_calendar/templates/activity_calendar/activity_page_base.html

```diff
@@ -141,7 +132,12 @@ <h1>{{ activity.title }}</h1>
                                     Activity is open for sign-ups
                                 {% endif %}
                             {% else %}
-                                Subscriptions will open on {% get_opening_time "j E, H:i" %}
+                                {% opens_in_future as opens_in_future %}
```

Comment:
This seems like it shouldn't be there right?

Reply:
It's a template tag, might have been better as a filter, I recognise that. I think I'll address that in the next PR that uses ActivityMoment objects to take care of these kind of things